lib/github-command-upload

```diff
@@ -99,6 +99,10 @@ upload_for_release() {
         fi
         DISTS_VM="${allowed_dists% dom0}" \
         COMPONENTS="$component" \
+        "$local_builder_dir/scripts/make-with-log" \
+            "update-repo-$repo_name" || :
```

Comment:
Uhm, sadly this would not work. At least not without changes on signing vm side. In case of not uploading log before requesting a signature, it doesn't fail. It displays standard split-gpg confirmation prompt. And wait indefinitely. Holding lock on given qubes-builder instance :/